jib-maven-plugin/src/integration-test/java/com/google/cloud/tools/jib/maven/BuildImageMojoIntegrationTest.java

```diff
@@ -76,20 +76,12 @@ private static String getTestImageReference(String label) {
     return nameBase + label + System.nanoTime();
   }
 
-  static String assertImageDigest(Path projectRoot) throws IOException, DigestException {
-    Path digestPath = projectRoot.resolve("target/jib-image.digest");
+  static String assertDigestFile(Path digestPath) throws IOException, DigestException {
```

Comment:
Looking at this again, I think `readDigestFile()` sounds better than `assertDigestFile`.